wLabs/LabSetup.md

```diff
@@ -0,0 +1,595 @@
+
+* **[Step 1:](#step-1-setting-up-the-database)** Download the database and install locally
+* **[Step 2:](#step-2-setting-up-the-data-connection)** Set up the Data Connections on the Interject Portal site
+* **[Step 3:](#step-3-setting-up-the-data-portals)** Set up the Data Portals on the Interject Portal site
+* **[Step 4:](#step-4-setting-up-fincube)** Set up the FinCube Portals
+* **[Step 5:](#step-5-the-report-files)** Download the lab files
+* **[Step 6:](#step-6-renaming-the-data-portals)** Change the Data Portal names in the files to match the ones manually set up
+
+The following walks through the steps to set up a Lab environment locally.
+
+### Step 1: Setting up the Database
+
+The first step in manually setting up a Lab environment is to establish a database on a local system where the Lab Guide will be used. Interject's choice of database is [SQL Server](https://www.microsoft.com/en-us/sql-server/sql-server-downloads){:target="_blank"}{:rel="noopener noreferrer"} as the Lab Guide relies on Microsoft's [Northwind Database](https://learn.microsoft.com/en-us/dotnet/framework/data/adonet/sql/linq/downloading-sample-databases){:target="_blank"}{:rel="noopener noreferrer"}.
+
+The following is a minimalist script to recreate the database using SQL Server 2014 or greater:
+
+[labs_db_script_sql_server_2014.sql][1]
+
+[1]:{{ site.url }}/download/labs_db_script_sql_server_2014.sql
+
+This script will have all the necessary tables, functions, procedures, schemas, and roles. You will need to first create the database of your choosing and then run this script inside that database. You may use [SSMS](https://docs.microsoft.com/en-us/sql/ssms/download-sql-server-management-studio-ssms?view=sql-server-2017){:target="_blank"}{:rel="noopener noreferrer"} or another editor of your choice.
+
+### Step 2: Setting up the Data Connection
+
+Navigate to the [Interject Portal site](https://portal.gointerject.com/) and log in. For a guide about how to use the Interject Portal site, see [here](/wPortal/The-INTERJECT-Website-Portal.html).
+
+Set up a Data Connection that points to the database you set up. The following example uses a database called "Interject_Lab_DB":
+
+* **Name:** Local_Lab_DataSource
+* **Description:** This is the Data Connection to a local database set up for the Lab Guide
+* **Connection Type:** Database
+* **Connection String:** Data Source=(local);Initial Catalog=Interject_Lab_DB;Integrated Security=SSPI;
+
+![](/images/LabSetup/DataConnection.png)
+<br>
+
+For detailed instructions on how to set up a Data Connection, see [Portal: Database Connection](/wPortal/L-Database-Connection.html).
+
+### Step 3: Setting up the Data Portals
+
+Now that you have a database and set up a Data Connection to it, you need to start adding Data Portals. For instructions on how to set up Data Portals, see [Data Portals](/wPortal/Data-Portals.html). Following is a list of the Data Portals and their Stored Procedures used in the Labs.
```

Comment:
im confused here. the instructions say we need to add data portals but we only show the SP name and params? shouldnt we have a script that sets up all data portals like we provided above to recreate the northwind database? that seems like the easiest way for users to get setup with SPs for portals